python/cuml/tests/test_base.py

```diff
@@ -24,6 +24,7 @@
 from cuml.testing.utils import \
     get_classes_from_package, \
     small_classification_dataset
+from cuml.experimental.common.base import Base as experimentalBase
```

Comment:
Should we just rename this symbol within the module here instead of the alias import?